keps/sig-instrumentation/3077-contextual-logging/kep.yaml

```diff
@@ -0,0 +1,39 @@
+title: Contextual logging
+kep-number: 3077
+authors:
+  - "@pohly"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-scheduling
+status: implementable
+creation-date: 2021-12-06
+reviewers:
+  - "@alculquicondor"
+  - "@liggit"
+  - "@serathius"
+approvers:
+  - "@ehashman"
```

Comment:
needs at least one [klog owner](https://raw.githubusercontent.com/kubernetes/klog/master/OWNERS) as approver (suggest @thockin and/or @dims, per https://github.com/kubernetes/enhancements/pull/3078/files#r797743432)

Reply:
please add @thockin here as approver, this is on his queue for today.

Reply:
Done.